### PR TITLE
Added support for IPv4 CIDR notation

### PR DESCRIPTION
**Pull request by jotadepicas**
Added cidr-js dependency and corresponding test cases.

### CODE REVIEW

**Comment by jotadepicas** [comment]
security checks have failed, will fix them and resubmit the PR. (checks failed because of cidr-js third-party dependencies).

**Comment by jotadepicas** [comment]
Replaced cidr dependency with netmask module, which does not have security issues. As a bonus: netmask handles base and broadcast addresses better.

File: index.js [hunk 1]
```diff
@@ -5,8 +5,13 @@ var once = require('once')
...
 var zlib = require('zlib')
 
+var Netmask = require('netmask').Netmask
+
 /** this regex will math both IP ranges and single IPs, with or without a description */
-var blocklistRe = /^(?:(\s*[^#].*?)\s*:\s*)?([a-f0-9.:]+?){1}(?:\s*-\s*([a-f0-9.:]+?)\s*)?$/
+var ipSetRegex = /^(?:(\s*[^#].*?)\s*:\s*)?([a-f0-9.:]+?){1}(?:\s*-\s*([a-f0-9.:]+?)\s*)?$/
+
+/** this regex matches IPv4 ranges in the form A.B.C.D/E, with or without a description */
+var ipv4NetSetRegex = /^(?:(\s*[^#].*?)\s*:\s*)?([0-9.:]+?)\/([0-9]{1,2}){1}\s*$/
```
Comment (feross) [inline index.js]: Unneeded `{1}`.

File: index.js [hunk 2]
```diff
@@ -36,11 +41,26 @@ module.exports = function loadIPSet (input, opts, cb) {
...
   }
+
+  function parseIPRange (regexMatch) {
+    var ip = regexMatch[2]
+    var bitMask = regexMatch[3]
+
+    var ipRange = ip + '/' + bitMask
+    var range = new Netmask(ipRange)
+
+    return {start: range.first, end: range.last}
```
Comment (feross) [inline index.js]: we should use `range.broadcast` because that's what this library is calling the real last IP.

**Comment by feross** [comment]
Released as 1.3.0

**Pull request merged**